MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Defaults.xaml

```diff
@@ -28,6 +28,7 @@
         <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.ScrollViewer.xaml" />
         <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Shadows.xaml" />
         <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Slider.xaml" />
+        <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Tabs.xaml" />
```

Comment:
I don't think we want the tab changes as part of this PR. Let me know if you need any help separating them.

Reply:
Right. Actually I've made some confusion on changing files without change the active branche. I'll try to resolve. Maybe I'll need help here.